api/conftest.py

```diff
@@ -966,9 +967,22 @@ def flagsmith_environments_v2_table(dynamodb: DynamoDBServiceResource) -> Table:
 
 
 @pytest.fixture()
-def feature_external_resource(feature: Feature) -> FeatureExternalResource:
+def feature_external_resource(
+    feature: Feature, mocker: MockerFixture
+) -> FeatureExternalResource:
+    from tests.unit.features.test_unit_feature_external_resources_views import (
+        mocked_requests_post,
+    )
+
+    mock_generate_token = mocker.patch(
+        "integrations.github.client.generate_token",
+    )
+    mock_generate_token.return_value = "mocked_token"
+
+    mocker.patch("requests.post", side_effect=mocked_requests_post)
```

Comment:
Where is `mocked_requests_post` coming from?

Reply:
Converted into a fixture in conftest